base/audit/events.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2024-Present Couchbase, Inc.
+//
+// Use of this software is governed by the Business Source License included
+// in the file licenses/BSL-Couchbase.txt.  As of the Change Date specified
+// in that file, in accordance with the Business Source License, use of this
+// software will be governed by the Apache License, Version 2.0, included in
+// the file licenses/APL2.txt.
+
+package audit
+
+const (
+	// auditdSyncGatewayStartID is the start of an ID range allocated for Sync Gateway by auditd
+	auditdSyncGatewayStartID ID = 53248
+
+	IDPlaceholder ID = 54000
```

Comment:
Are we going to define events up from this number? I wonder if it is worth doing for self documenting

```
	// auditdSyncGatewayStartID is the start of an ID range allocated for Sync Gateway by auditd
	auditdSyncGatewayStartID ID = iota + 53248

    // IDPlaceholderID is the last expected event and used for testing
	IDPlaceholder ID = 54000
```

Reply:
Not sure yet. That'll come later once the event table is finalised.

I wanted to avoid iota so we had searchable IDs in the codebase, rather than rely on the auto-incrementing iota value